### ISSUE TITLE
Subject show method needs fixing?

### ISSUE THREAD

**Issue by vjd** [PumasAI/Pumas.jl#491]
https://github.com/UMCTM/PuMaS.jl/issues/443 was closed a few days ago. However, the show method is not showing covariates by default now, although the treeview in Juno shows it. 

```
julia> s1 = Subject(id=1,evs=ev,cvs=(isPM=0, Wt=70))
Subject
  ID: 1
  Events: 1
```

The treeview has all the information -
![image](https://user-images.githubusercontent.com/1425562/57441162-02760e00-7267-11e9-98f1-437ce4b87306.png)

**Comment by logankilpatrick** [comment]
I added this back in and it works @vjd . Per @ChrisRackauckas comment, should I put in a limit on the amount of text that gets shown and then a simple "The covariant is too long to display" message if it's over the max length?

**Comment by logankilpatrick** [comment]
```
julia> s1 = Subject(id=1,  evs=ev, cvs=(isPM=1, wt=70))
Subject
  ID: 1
  Events: 5
  Covariates: (isPM = 1, wt = 70)
```

**Comment by vjd** [comment]
Yeah, truncated printing with a useful message is fine. Perhaps tell the users to call `DataFrame(s1)` for a table view of the subject instead.

**Comment by logankilpatrick** [comment]
What would be a high number of Covariates?

**Comment by vjd** [comment]
lets cut at 10

**Comment by logankilpatrick** [comment]
Closed with #691
